package/mash-client.spec

```diff
@@ -26,6 +26,7 @@ URL:            https://github.com/SUSE-enceladus/mash-client
 Source:         https://files.pythonhosted.org/packages/source/p/mash-client/%{name}-%{version}.tar.gz
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
+BuildRequires:  python3-rpm-macros
```

Comment:
Please use `python-rpm-macros`. `python3-rpm-macros` is just a `Provides` and as such if someone searches for `python3-rpm-macros` in the Build Service nothing will show up which will lead to an unnecessary goose chase.

Reply:
Rebased